src/tests/ftest/deployment/target_failure.yaml

```diff
@@ -0,0 +1,72 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+
+timeout: 10M
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.2"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+
+pool_size_ratio:
+  size: 50%
+  control_method: dmg
+pool_size_value:
+  scm_size: 100G
+  nvme_size: 300G
+  control_method: dmg
+
+container_wo_rf:
+  type: POSIX
+  control_method: daos
+container_with_rf:
+  type: POSIX
+  control_method: daos
+  properties: rf:1
+
+ior:
+  client_processes:
+    np: 4
+  iorflags:
+    flags: "-k -D 10 -v -w -W"
+    api: DFS
+    # We wait for a few seconds before excluding a target. If the IOR finishes too
+    # quickly, the test will not work. We use -D (deadlineForStonewalling) 10, so the
+    # process ends in about 10 sec. The following transfer_size - block_size combination
+    # takes about 42 sec in the Shared Cluster. Adjust the value based on the system so
+    # that it takes longer than the -D value.
+    transfer_size: '512K'
+    block_size: '30G'
```

Comment:
If you're relying on stonewall, you could just just set block_size to something really high like `150G`, which wouldn't need adjustment. And IOR will never actually write this much data

Reply:
Actually we try running IOR after reintegrating the target, so we'll write this much data. I increased it to 150G, but it works in CI, so it's fine for now.

Reply:
IOR only writes up to the stonewall time OR up to the aggregate block size - whichever comes first. For example:
`mpirun -np 1 -b 150G -k -w -D 1`
Will write for only 1 second, not 150G. Unless the write bandwidth is greater than 150G!
Anyway, what you have works so we can stick with that for now.